src/convertToObject.js

```diff
@@ -12,7 +12,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittingSource = sourceString.split('\n');
+  const unvalidProperties = splittingSource
+    .map(property => property.split(';'));
+  const filteredProperties = unvalidProperties
+    .filter(property => property[0].length > 3);
+  const unionFiltered = filteredProperties
+    .map(property => property.join('').split(':'));
+  const validProperties = unionFiltered
+    .map(value => [value[0].trim(), value[1].trim()]);
```

Comment:
Роби відступи між змінними
```suggestion
  const splittingSource = sourceString.split('\n');
  
  const unvalidProperties = splittingSource
    .map(property => property.split(';'));
    
  const filteredProperties = unvalidProperties
    .filter(property => property[0].length > 3);
    
  const unionFiltered = filteredProperties
    .map(property => property.join('').split(':'));
    
  const validProperties = unionFiltered
    .map(value => [value[0].trim(), value[1].trim()]);
```